docker-compose.yml

```diff
@@ -582,6 +584,19 @@ services:
   #     - './debug:/tmp'
   #   ports:
   #     - '3001:80'
+
+  live-reloader:
+    image: apogiatzis/livereloading
+    container_name: livereloader
+    privileged: true
+    environment:
```

Comment:
I'd like to use some additional vars from this project:
`RELOAD_DELAY` = 1 sec (Default 1.5, but it's nice to see the number)
`RESTART_TIMEOUT` = 1 sec (Not all our containers respond to `SIGINT` and only die when `SIGKILL` is sent after the timeout of 10 sec)

Reply:
Sure, I'll add the number so you can see it :)
Ah alright I probably would've noticed this if I had added it to all the containers, but will do!